src/recordlinker/linking/link.py

```diff
@@ -105,13 +105,12 @@ def compare(
         results.append(result)
     return matching_rule(results, **kwargs)  # type: ignore
 
-
 def link_record_against_mpi(
-    record: dict,
+    record: schemas.PIIRecord,
```

Comment:
If we change this to be a PIIRecord, what happens when a user invokes the `/link-record` endpoint?

Reply:
oh nevermind, I see line 197 now